pymemcache/client.py

```diff
@@ -113,11 +114,13 @@ def json_deserializer(key, value, flags):
 
 
 class MemcacheError(Exception):
+
     "Base exception class"
     pass
 
 
 class MemcacheClientError(MemcacheError):
+
```

Comment:
Please remove these extra empty lines in the Error classes (see the five below).